### PR TITLE
Readd Windows Python 3.8 (cont.)

### PR DESCRIPTION
**Pull request by mbargull**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

continuation of gh-40

### CODE REVIEW

**Comment by mbargull** [comment]
I think the added patch is fine as a temporary workaround. An alternative might've been to use `CDLL(..., winmode=...)` to restore pre-3.8 behavior, but I don't think we have to do that.
Ideally, the actual fix is probably one of:
- Patch the `python` package so that `%LIBRARY_BIN%` is always considered.
- Let CMake add a manifest with `%LIBRARY_BIN%` added as a search path. I don't know enough about CMake and Windows library loading to do that, though.
  ^- That is assuming that manifest can add RPATH equivalents, which I don't actually know. Anyone else know more?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -56,3 +56,4 @@ extra:
     - souravsingh
     - marcelotrevisani
     - xhochy
+    - mbargull
```
Comment (mbargull) [inline recipe/meta.yaml:59]: I forgot that I hadn't added myself here, but only on the Numba feedstock :sweat_smile:.

**Comment by mbargull** [comment]
> Patch the `python` package so that `%LIBRARY_BIN%` is always considered.

We already have: https://github.com/conda-forge/python-feedstock/blob/d7fa9190941c6505c3be86ff15e762988b743ee5/recipe/patches/0022-Add-CondaEcosystemModifyDllSearchPath.patch
So I don't know why that didn't suffice in this case.
@conda-forge/python:
What we had here is a DLL (`llmvlite.dll`) which itself does not lie in `%LIBRARY_BIN%` but is linked to `zlib.dll` which is in `%LIBRARY_BIN%`, of course. Attempting to load `llvmlite.dll` with its full path did not succeed on Win+Py3.8 until we added `os.add_dll_directory(os.path.join(os.path.normpath(sys.prefix), 'Library', 'bin'))`.
I would assume that `0022-Add-CondaEcosystemModifyDllSearchPath.patch` would've taken care of this already. Do any of you know what's up with this?

**Comment by jakirkham** [comment]
Thanks Marcel! 😄 This seems like a good solution to me. Thoughts from others (cc @conda-forge/numba )? 🙂

**Comment by jakirkham** [comment]
Going to go ahead and merge to unblock the `numba` release. Though if anyone has thoughts on this, please do let us know 🙂

**Comment by jakirkham** [comment]
Linking the relevant upstream `python` issue ( https://github.com/conda-forge/python-feedstock/issues/307 ).

**Pull request merged**